src/backend/cuda/cu_handles.hpp

```diff
@@ -0,0 +1,20 @@
+/*******************************************************
+ * Copyright (c) 2019, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#pragma once
+
+#include <cublas_v2.h>
+#include <cusolverDn.h>
+#include <cusparse_v2.h>
+
+namespace cuda {
+using BlasHandle = cublasHandle_t;
```

Comment:
Why not just define them in the cublas.hpp file like earlier? These are handles from distinct libraries why mix them if you don't have to?

Reply:
These handles are `cu*` libraries. I can declare them like I did `clfft` stuff, let me check.